src/pages/components/menu-buttons/accessibility.mdx

```diff
@@ -65,22 +131,81 @@ with focus (which causes focus to go somewhere else and the menu to close).
 ![illustration showing an open menu with the focus on the first item, and the arrow and Esc keys called out](images/overflow-menu-accessibility-2.png)
 
 <Caption>
-  When opened, the first item in the menu takes focus. Arrow keys move focus,
+  When opened, the first item in the menu takes focus. Arrow keys move focus and
   Esc closes the menu.
 </Caption>
 
 </Column>
 </Row>
 
+## Design recommendations
+
+Design annotations are needed for the following instances.
+
+### Labeling
+
+#### Combo button
+
+Combo button has a prop that can be used to change the tooltip label and is only
+intended to be changed to translate _'Additional actions'_ into other languages.
+If translation is required, the translation must be annotated with a label that
+is exposed on hover or focus.
+
+<Row>
+<Column colLg={8}>
+
+![a combo button which is a part of a translated page that has an annotation 'label=अतिरिक्त कार्रवाइयां'](images/combo-button-accessibility-3.png)
+
+<Caption>
+  Annotate the label for combo button so the proper tooltip appears, which
+  should only be used for translation purposes.
+</Caption>
+
+</Column>
+</Row>
+
+#### Overflow menu
+
+Overflow menu’s
+[custom icon variant](https://react.carbondesignsystem.com/?path=/story/components-overflowmenu--render-custom-icon)
+has a prop that can be used to change the tooltip label. If the tooltip label is
+changed, they must be annotated with a label that will be exposed on hover or
+focus.
+
+<Row>
+<Column colLg={8}>
+
+![a custom icon over flow menu with a share icon that has an annotation 'label=sharing options'](images/overflow-menu-accessibility-3.png)
```

Comment:
This image needs to replace the overflow menu icons for the the first two cards with the share icon, to be consistent with the last card that has the share icon.